app/src/main/java/com/sdp13epfl2021/projmag/notification/MyFirebaseMessagingService.kt

```diff
@@ -0,0 +1,74 @@
+package com.sdp13epfl2021.projmag.notification
+
+
+import android.app.NotificationChannel
+import android.app.NotificationManager
+import android.app.NotificationManager.IMPORTANCE_HIGH
+import android.app.PendingIntent
+import android.app.PendingIntent.FLAG_ONE_SHOT
+import android.content.Context
+import android.content.Intent
+import android.content.SharedPreferences
+import android.os.Build
+import androidx.annotation.RequiresApi
+import androidx.core.app.NotificationCompat
+import com.google.firebase.messaging.FirebaseMessagingService
+import com.google.firebase.messaging.RemoteMessage
+import com.sdp13epfl2021.projmag.MainActivity
+import com.sdp13epfl2021.projmag.R
+import kotlin.random.Random
+
+private const val CHANNEL_ID = "my_channel"
+class MyFirebaseMessagingService: FirebaseMessagingService() {
```

Comment:
`MyFirebaseMessagingService` is not very self explanatory. Maybe `ProjectNotificatonService` or something more related to the code that you wrote